.github/workflows/codeql-analysis.yml

```diff
@@ -64,4 +64,4 @@ jobs:
     #   make release
 
     - name: Perform CodeQL Analysis
-      uses: github/codeql-action/analyze@v1
\ No newline at end of file
+      uses: github/codeql-action/analyze@v3
```

Comment:
Can you please add a newline at the end of the file? I forgot it in the first place...